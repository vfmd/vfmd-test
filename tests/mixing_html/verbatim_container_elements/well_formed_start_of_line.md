## pre

<pre> Some text

### Inside pre

</pre> Some text

## style

<style> Some text

### Inside style

</style> Some text

## script

<script> Some text

### Inside script

</script> Some text 
