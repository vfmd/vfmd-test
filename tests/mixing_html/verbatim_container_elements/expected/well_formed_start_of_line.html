<h2>pre</h2>
<pre> Some text

### Inside pre

</pre> Some text

<h2>style</h2>
<style> Some text

### Inside style

</style> Some text

<h2>script</h2>
<script> Some text

### Inside script

</script> Some text